src/frontend/src/utils/statusList/statusList.ts

```diff
@@ -10,9 +10,9 @@ import {
 } from './statuses/index';
 import { StatusList } from './types';
 
-export const statusList = (pia: IPiaForm | null): StatusList => {
+export const statusList = (pia: IPiaForm | null, from?: string): StatusList => {
```

Comment:
Can you incorporate this functionality into src\frontend\src\utils\statusList\common.ts